plugins/ssh-plugin/package.json

```diff
@@ -32,6 +32,7 @@
         "@eclipse-che/plugin": "0.0.1"
       },
       "devDependencies": {
+        "@types/node": "^10.17.21",
```

Comment:
Why do we need a different version here?

Reply:
in any case all 10.0.0 resolve to 10.17.21 or later